Makefile

```diff
@@ -126,10 +126,19 @@ $(foreach lib,$(VALID_LIBRARIES),$(call include_lib,$(lib)))
 # Includes all projects so make can find their targets
 $(foreach proj,$(VALID_PROJECTS),$(call include_proj,$(proj)))
 
+IGNORE_CLEANUP_LIBS := CMSIS STM32F0xx_StdPeriph_Driver stm32f0xx unity
+FIND_PATHS := $(addprefix -o -path $(LIB_DIR)/,$(IGNORE_CLEANUP_LIBS))
+FIND := find $(PROJ_DIR) $(LIB_DIR) \
+			  \( $(wordlist 2,$(words $(FIND_PATHS)),$(FIND_PATHS)) \) -prune -o \
+				-name "*.[ch]" -print
+
 # Lints the files in ms-common and projects
 lint:
-	@find $(PROJ_DIR) -name "*.c" -o -name "*.h" | xargs -P 24 -r python2 lint.py
-	@find "$(LIB_DIR)/ms-common" -name "*.c" -o -name "*.h" | xargs -P 24 -r python2 lint.py
+	@$(FIND) | xargs -r python2 lint.py
+
+format:
+	@echo "Formatting *.[ch] in $(PROJ_DIR), $(LIB_DIR)"
```

Comment:
I don't format them - That's what `IGNORE_CLEANUP_LIBS` is for. I was just lazy in the echo. :P